### PR TITLE
23/08 Lunchtime pull

### PR DESCRIPTION
**Pull request by NyoomNyoom**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,3 +6,5 @@ Our github page for COSC345 2022. Our project is called Manu and is going to be
 Our team consists of Daniel Robinson(@Daniel-Ian-Robinson), Will Rushton(@wrushton0), Madeline McCane(@mgmccane), and Jackson North(@NyoomNyoom).
 
 For the most up to date version of our quiz game, please clone the "main" branch as the "Stage_" branch is the branch that we will be working out of. The main branch of our code is updated weekly every Monday and Friday from our progress throughout the week.
+
+A feature we are testing is gesture based navigation between the menu and infographic, this feature hasn't been brought to full functionality. However, it is important to us to gain feedback about this function.
```
Comment (Daniel-Ian-Robinson) [inline README.md:10]: Perfect! Thank you Will.

File: ManuProjectFiles/app/src/main/res/layout/info_graphic_activity.xml [hunk 2]
```diff
@@ -11,7 +11,7 @@
         android:layout_width="match_parent"
         android:layout_height="match_parent"
         android:scaleType = "centerCrop"
-        android:src = "@drawable/fern">
+        android:src = "@drawable/fern_alt">
```
Comment (NyoomNyoom) [inline ManuProjectFiles/app/src/main/res/layout/info_graphic_activity.xml:16]: Is this the darker fern photo?

**Comment by Daniel-Ian-Robinson** [comment]
Looks good apart from my comment in cf1acf3.

File: ManuProjectFiles/app/src/main/res/layout/info_graphic_activity.xml (hunk 2 above)
Comment (wrushton0) [reply ManuProjectFiles/app/src/main/res/layout/info_graphic_activity.xml:16]: yes it is

**Comment by wrushton0** [comment]
Looks decent to me, nothing complicated added

**Comment by Daniel-Ian-Robinson** [comment]
ManuProjectFiles/app/src/main/res/layout/quiz_options.xml lines 30, 37, 44

and the corresponding lines in,

ManuProjectFiles/app/src/main/java/com/example/manu/QuizOptions.kt lines 29, 34, 40

To keep with the convention, use "btn_eng". This is because the view type is a button and we have no obvious data type in Kotlin, hence the variable name is the view's full descriptor.

File: ManuProjectFiles/app/src/main/res/layout/info_graphic_activity.xml (hunk 2 above)
Comment (NyoomNyoom) [reply ManuProjectFiles/app/src/main/res/layout/info_graphic_activity.xml:16]: Sweet

**Pull request merged**